php_scrypt.c

```diff
@@ -199,11 +199,11 @@ PHP_FUNCTION(scrypt)
 		php_hash_bin2hex(hex, buf, keyLength);
 		efree(buf);
 		hex[keyLength*2] = '\0';
-		RETURN_STRINGL(hex, keyLength * 2);
+		RETVAL_STRINGL(hex, keyLength * 2);
```

Comment:
@remicollet I'd appreciate your review as well :)

Reply:
LGTM

BTW, double allocation could be avoid, using something like https://github.com/php/php-src/blob/master/ext/hash/hash.c#L411

Reply:
Thank you for the suggestion! BTW: I've already increased the minimum version requirement to PHP 7.0 yesterday :)